jsonrpc/eth_endpoint.go

```diff
@@ -147,18 +148,11 @@ func (e *Eth) SendRawTransaction(input string) (interface{}, error) {
 	return tx.Hash.String(), nil
 }
 
-// SendTransaction creates new message call transaction or a contract creation, if the data field contains code.
+// Reject eth_sendTransaction json-rpc call as we don't support wallet management
 func (e *Eth) SendTransaction(arg *txnArgs) (interface{}, error) {
-	transaction, err := e.d.decodeTxn(arg)
-	if err != nil {
-		return nil, err
-	}
-
-	if err := e.d.store.AddTx(transaction); err != nil {
-		return nil, err
-	}
+	errMsg := "wallet management is not supported, any request call to the eth_sendTransaction method is not supported"
```

Comment:
You can inline this error message with the return

Reply:
FIxed in commit cbc116c8211dc3ecd4685616b182a8d6db4cb005